jupyter-on-gke/README.md

```diff
@@ -70,7 +77,14 @@ We allow user to set their own domains, in the `variables.tf` file. Since we are
 
 9. Run `terraform apply`
 
-## Authentication (Currently Enabled By Default)
+## Securing your Jupyter Endpoint
+
+For demo purposes, disabling auth creates a public IP for the Jupyter notebook with basic dummy authentication. To secure your cluster, it is _strongly recommended_ to have auth enable and use IAP.
```

Comment:
"To secure the Jupyter endpoint, this example enables IAP by default. It is strongly recommended to keep this configuration. If you wish to disable it, do the following: set XYZ flag to false in XYZ file."

Reply:
Done!